src/services/books/saga.ts

```diff
@@ -21,7 +21,7 @@ function* isAvailableAtSelect(bIds: string[]) {
   try {
     const books: BooksState = yield select((state: RootState) => state.books);
     const availableBIds = bIds.filter(
-      (bId) => books.items[bId] && !books.items[bId].clientBookFields?.isAlreadyCheckedAtSelect,
+      (bId) => books.items[bId] && !books?.items[bId]?.clientBookFields?.isAlreadyCheckedAtSelect,
```

Comment:
```suggestion
      (bId) => books.items[bId] && !books.items[bId].clientBookFields.isAlreadyCheckedAtSelect,
```

처음 `books.items[bId]`에서 에러가 안 났단 건 뒤에 있는 `books`에도 `?`를 붙일 필요가 없다는 뜻이겠죠?

Reply:
오우 이 부분은 옵셔널 체이닝을 때버리면 `Object is possibly 'null'.` 가 뜨네요!?

Reply:
아 타입스크립트가 또... 전에 본 거랑 똑같은 케이스네요. 함수 본문에서 `books.items[bId]`를 변수로 빼는 게 좋아보여요